README.md

```diff
@@ -1,60 +1,49 @@
-new (expanded) region and direct API traffic to that new instance. Then, you can drain down the existing
-instance, delete it, and recreate it in the same region as the one you deleted.
-
-Apigee has provided a set of scripts in this project that perform all of the required steps to recreate an
-instance. 
-
-The basic steps are:
+On January 24, 2022, Apigee introduced an enhancement to address this problem.
+The enhancement reduces the IP range required to peer your VPC network with
+Apigee and uses privately used public IPs (PUPI) to allow workloads to scale to
+higher limits.
 
-1. Update a configuration script, `source.sh`.
-2. Create, configure, and direct API traffic to a new, temporary Apigee instance in a new region.
-3. Delete the original instance (the instance you are replacing).
-4. Create, configure, and direct API traffic to a new instance in the same region as the original instance.
-5. Delete the temporary instance.
+This project provides multiple options to recreate Apigee instance in order to
+address the IP issue and take advantage of PUPI with Apigee.
 
-### 1. Update `source.sh`
+*   ***[Recreate instance with no downtime and no data loss](./recreate_with_no_downtime_and_no_data_loss.md)***:
+    This is the recommended approach. Ideal if you are already multi region
+    Apigee but are not okay with other regions handling API requests temporarily
+    and don't want any data loss or downtime. This requires you to create a new
+    Apigee Instance.
 
-This script defines a templated function that you must copy and modify. When you are finished, the script will have
-two functions: one with parameter values for the existing instance (the one you are replacing),
-and one for a new, temporary instance. The scripts you will run later call these functions to perform
-their tasks.
-
-The basic steps are:
-
-1. Fill in your Google Cloud project ID at the top of the `source.sh` script.
-2. Copy the function block.
-3. Change the name of the first function to: `INIT_REGION_${REGION_ID}`, where `REGION_ID` is
-the name of the region in which you will create a temporary instance. This region cannot be
-the same as the region in which your existing instance is provisioned. For example, if
-the new region is `us-east1`, rename the function:
+*   ***[Recreate instance with downtime and data loss - reuse other existing
+    instances](./recreate_with_no_downtime_and_no_data_existing_multi_region.md)***:
+    This is the recommended approach if you are already multi region Apigee and
+    are okay with other regions handling all the API requests temporarily and
+    don't want any data loss and downtime.
 
-    `INIT_REGION_US_EAST1`
+*   ***[Recreate instance with downtime but with no data loss](./recreate_with_downtime_and_no_data_loss.md)***:
+    This is good if you are okay with downtime(probably for dev or qa Apigee
+    org) but still want to retain cassandra data.
```

Comment:
consider reword:  This is good if you are okay with downtime (probably suitable for non-production Apigee organizations) but still want to retain the runtime data.